src/obr/signac_wrapper/operations.py

```diff
@@ -219,7 +220,9 @@ def execute_operation(job: Job, operation_name: str, operations) -> Literal[True
                 func = list(operation.keys())[0]
                 getattr(sys.modules[__name__], func)(job, operation.get(func))
         except Exception as e:
+            tb = traceback.format_exc()
             logging.error(e)
+            logging.info(tb)
```

Comment:
I would personally prefer to have the error printed as the most recent output of a program. 
Purely for the sake of not having to scour the command line output for the actual error under a bunch of stack traces.
```suggestion
            logging.info(tb)
            logging.error(e)
```